alerting/ingest-alerting/variables.tf

```diff
@@ -0,0 +1,32 @@
+# Global
+
+variable "suffix" {
+  description = "Optional suffix to identify and avoid duplication of unique resources"
```

Comment:
It is only to identify in this specific case because there is no possible duplication here